server/src/clang-utils/SourceToHeaderMatchCallback.cpp

```diff
@@ -311,6 +317,9 @@ void SourceToHeaderMatchCallback::print(const NamedDecl *decl, const PrintingPol
     }
     auto name = decl->getNameAsString();
     auto decoratedName = decorate(name);
+    if (beginsWithFloat(decoratedName)) {
+        decoratedName = decoratedName.substr(1, decoratedName.length() - 1);
```

Comment:
If you want to make a substring from the position to the end, you shouldn't use the second argument in the function `substr`.